mu4/notation/inotationcontextmenu.h

```diff
@@ -0,0 +1,36 @@
+//=============================================================================
+//  MuseScore
```

Comment:
Better to move this class (and related classes) to internal of the module

Reply:
thanks, fixed